server/game/cards/01-Core/SelwynTheFence.js

```diff
@@ -5,7 +5,8 @@ class SelwynTheFence extends Card {
         this.fight({
             reap: true,
             target: {
-                cardType: 'creature',
+                activePromptTitle: 'Choose a captured aember to move to your pool.',
```

Comment:
If you use 'amber' it will automatically be converted to Aember and a amber icon by the client